go.mod

```diff
@@ -1,10 +1,10 @@
-module github.com/google/go-github/v28
+module github.com/google/go-github/v29
 
 require (
 	github.com/golang/protobuf v1.3.2 // indirect
 	github.com/google/go-querystring v1.0.0
 	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2
-	golang.org/x/net v0.0.0-20190311183353-d8887717615a // indirect
+	golang.org/x/net v0.0.0-20191014212845-da9a3fd4c582 // indirect
```

Comment:
This change doesn’t look intentional.

I suspect it came about from the earlier version of this CL where the scrape/go.mod file was removed, and its requirements merged with that of the main go-github module. The scrape module happened to require a newer version of x/net:

https://github.com/google/go-github/blob/04a52c26bbdd106ed23331c4d61fde482ffb9413/scrape/go.mod#L10

And so it stayed around after `go mod tidy` because this module also needs x/net.

It’s not harmful to keep it since it’s just a newer x/net, but it’s also probably better to make changes intentionally rather than due to chance.

You can go back to original or stay with the newer, I’m okay with both.

Reply:
Ah, I see.  Thank you, @dmitshur!
Reverted.